### DISCUSSION TITLE
How Can I use additional maps in "Quick start" Latest version

### DISCUSSION THREAD

**Discussion by cpsvt** [carla-simulator/carla#4411] [General]
I have been trying to use Town06 and Town07 in the latest version of the quick start installation. If anyone wants to download the latest version through quick start, the latest version does NOT include any of the additional maps. In order to add Town06 or Town07 to the quick start version, I installed 9.11 version instead through the Windows built. I added all the content folder including the maps, nav etc. However, I've faced the following error. 
Now, I'm wondering how I can add these two to the quick start?

[
![Capture](https://user-images.githubusercontent.com/86995899/126005084-dc8acd1f-dae9-4ee4-9ac3-01a83931257e.PNG)
](url)

**Comment by corkyw10** [comment]
Hello, did you follow the quickstart documentation? It explains how to get the extra towns in the ["Import additional assets"](https://carla.readthedocs.io/en/0.9.11/start_quickstart/#import-additional-assets) section and it doesn't require using the windows build. It also looks like you are mixing versions 0.9.10 and 0.9.11 from looking at your command line paths.

**Reply by cpsvt** [reply]
@corkyw10  Thanks for you response.
Yes, I followed the documentation and that's exactly my question comes from. [ There is no Town06, Town07 for 0.9.11 ](https://github.com/carla-simulator/carla/blob/master/Docs/download.md). The nightly build is the latest version and it does not contain those towns. My question is, how can I add those towns to latest version (nightly build)?

Yes, I tried to copy and past Town06 form 0.9.10 to 0.9.11 and it did not work.

**Reply by corkyw10** [reply]
That link should be corrected in the next version of the docs, but in the meantime, you can check the [release](https://github.com/carla-simulator/carla/releases/tag/0.9.11) section of GitHub. Here you have the additional maps for 0.9.11. Mixing between versions will not work.

**Reply by cpsvt** [reply]
@corkyw10  Thanks a lot for your response.
I actually made some changes to one of the maps using UE4 Editor. Running it on my local pc takes significant CPU and GPU whereas the quick start version takes about half of it. I also disabled "Use less CPU when in background", but nothing has changed. For that reason, I was wondering whether there is a way that I can export the edited map and used it for the quick start version (both 0.9.11 ver.)?
Thanks for your response.

**Reply by corkyw10** [reply]
Yes you can do that by following the information [here](https://carla.readthedocs.io/en/latest/tuto_A_create_standalone/#export-a-package-in-a-carla-build-from-source). Even though this is the dev docs, the information is accurate for 0.9.11

**Reply by cpsvt** [reply]
> Yes you can do that by following the information [here](https://carla.readthedocs.io/en/latest/tuto_A_create_standalone/#export-a-package-in-a-carla-build-from-source). Even though this is the dev docs, the information is accurate for 0.9.11

Thanks a lot for your response @corkyw10.

I've made significant changes in one of the maps and I'm hoping to use it as a standalone package to make running the program faster. I followed the same procedure and executed **make package ARGS="--packages=RandomName"**  in the root directory where I normally execute **Make Launch** and it created a .tar.gz file, as can be seen below. 
 
However, the created folder is empty. I thought running this script creates a standalone package using all the created maps within the Unreal\CarlaUE4\Content\Carla\Maps  directory.  Should I create a package before executing the **make package**? I also wondering whether I need to define a path, because I see it gives the following error as the syntax is incorrect:
**/=\TARGET:/=\*
The filename, directory name, or volume label syntax is incorrect.**

 Thanks a lot for your help.

[
![Capture](https://user-images.githubusercontent.com/86995899/128263012-825fbc3d-31ed-46a7-8115-2dbc8953a4d4.PNG)
](url)
